### ISSUE TITLE
Ability to specify additional columns

### ISSUE THREAD

**Issue by greyblake** [albertosaurus/power_enum#4]
It would be great to have a migration syntax similar to this to add additional custom columns:

``` ruby
  create_enum :city, :description => true do |t|
    t.float :latitude
    t.float :longitude
  end

```

**Comment by albertosaurus** [comment]
Sounds reasonable.  I'll add this over the weekend.

**Comment by albertosaurus** [comment]
Implemented in version 0.5.0.
